recipe/recipe.yaml

```diff
@@ -0,0 +1,577 @@
+context:
+  version: "7.8.3"
+
+recipe:
+  name: suitesparse
+  version: ${{ version }}
+
+source:
+  url: https://github.com/DrTimothyAldenDavis/SuiteSparse/archive/v${{ version }}.tar.gz
+  sha256: ce39b28d4038a09c14f21e02c664401be73c0cb96a9198418d6a98a7db73a259
+
+build:
+  number: 2
+  string: ss${{ version | replace('.', '') }}_h${{ hash }}
+
+outputs:
+  - package:
+      name: libsuitesparseconfig
+    about:
+      license: BSD-3-Clause
+      license_file: LICENSE.txt
+    build:
+      script:
+        # rattler-build adds `.sh` or `.bat` based on platform
+        file: build-one
+        env:
+          SUBPKG_DIR: SuiteSparse_config
+    requirements:
+      build:
+        - ${{ compiler('c') }}
+        - ${{ stdlib('c') }}
+        - ${{ compiler('fortran') }}
+        - cmake
+        - {if: win, then: ninja, else: make}
```

Comment:
Personal opinion: I would just use ninja for all platforms as errors are better readable with it.